### ISSUE TITLE
Erreur récurrente dans les logs

### ISSUE THREAD

**Issue by ErenHistarion** [eveoniris/larpManager-php#474]
`myapp.CRITICAL: LogicException: The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller? (uncaught exception) at /home/eveoniri/larpmanager/vendor/symfony/http-kernel/HttpKernel.php line 153 {"exception":"[object] (LogicException(code: 0): The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller? at /home/eveoniri/larpmanager/vendor/symfony/http-kernel/HttpKernel.php:153)"} []`

J'imagine qu'on arrivera à nettoyer ça avec les diverses mises à jour de la rentrée d'automne.

**Comment by Gectou4** [comment]
en soit rien de dramatique

Si tu as plus de stack trace on serait plus efficace ou si tu as l'url de la page a laquelle l'erreur est lié.

Globalement c'est dans une action d'un ou de plusieurs controller qui a du être typé

publiction function actionUpdate(): Response

mais dans laquelle on ne rencontre pas de return $quelquechose
Surement car il n'y a rien à faire/retourné et que donc il n'y a rien de grave en soit

juste sa pourris les logs ^^' mais il seront bon de vérifier/clean au cas par cas

**Comment by ErenHistarion** [comment]
Une autre qui apparait de temps en temps : 
`[2023-06-21 18:40:08] myapp.CRITICAL: Doctrine\DBAL\Exception\NotNullConstraintViolationException: An exception occurred while executing 'INSERT INTO personnage_background (`text`, visibility, creation_date, update_date, personnage_id, user_id, gn_id, discr) VALUES (?, ?, ?, ?, ?, ?, ?, ?)' with params ["<p>Intrigues LH6<\/p>", "PRIVATE", "2023-06-21 18:40:07", "2023-06-21 18:40:07", 4307, 4064, null, "extended"]: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'gn_id' cannot be null (uncaught exception) at /home/eveoniri/larpmanager/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 103`

**Comment by Gectou4** [comment]
yep c'est exactement ça; j'ai fais un peu la chasse à ce genre de mini "bug" pendant le set up 
quasi tous les formulaires sont touché ^^'
